docs/release-notes/release-notes-0.14.0.md

```diff
@@ -127,6 +127,10 @@ you.
   to make it less likely that we retry etcd transactions and make the commit
   queue more scalable.
 
+* [Flatten the payment-htlcs-bucket](https://github.com/lightningnetwork/lnd/pull/5635)
+  in order to make it possible to prefetch all htlc attempts of a payment in one
+  DB operation.
```

Comment:
Depending on the decision we make concerning the potentially large DB transaction, we might need to add some more information here what the user can do if they run into problems.

Reply:
ptal